rust-runtime/aws-smithy-types/README.md

```diff
@@ -0,0 +1,5 @@
+# Fundamental Types for Smithy Services
```

Comment:
Some readmes have a title like this, and others use the package name. Which pattern do we want to use?

Reply:
when the crate name was clear I used the crate name but otherwise I added a new name. We could standardize on just using the exact crate name, but I don't have super strong feelings.